OfferSpace.BL/Models/RequestCustomerDiscusion.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OfferSpace.BL.Models
+{
+    public class RequestCustomerDiscusion : IEntity<long>
+    {
+        [Key]
+        [ForeignKey("RequestCustomer")]
+        public long Id { get; set; }
+
+        public long? RequestCustomerId { get; set; }
```

Comment:
it can't be nullable due to every discussion has a reference on customer request. Discussion without RequestCustomer can't exist at all.